src/main/java/de/unistuttgart/overworldbackend/controller/BookController.java

```diff
@@ -16,6 +16,8 @@
 @RequestMapping("/courses/{courseId}/worlds/{worldIndex}")
 public class BookController {
 
+    private static final List<String> writeAccess = List.of("lecturer");
```

Comment:
1. Should we perhaps name this variable `lecturerRole`?
2. Constants are written in upper case: `LECTURER_ROLE`
3. As you declare it in multiple classes: What about using an explicit helper class where you define it once?

Reply:
I think we shouldn't call ist lecturerRole because there might be other roles in the future who have the same rights(admin,...)